cogs/streams.py

```diff
@@ -55,6 +58,21 @@ def __init__(self, bot):
             await self.bot.say("Error.")
 
     @commands.command()
+    async def youtube(self, channelid: str):
+        """Checks if youtube stream is online"""
+        channelid = escape_mass_mentions(channelid)
+        online = await self.youtube_online(channelid)
+        if online is True:
+            await self.bot.say("https://gaming.youtube.com/channel/{} "
```

Comment:
Personally I think its better to make this into another command. This will avoid most problems that will arise and will be easier to understand for users.